src/identity/allowances/actions/thunks/index.ts

```diff
@@ -0,0 +1,38 @@
+// Libraries
+import {Dispatch} from 'react'
+
+// Actions
+import {
+  Actions as OrgCreationAllowancesActions,
+  setOrgCreationAllowances,
+  setOrgCreationAllowancesStatus,
+} from 'src/identity/allowances/actions/creators'
+
+// API
+import {fetchOrgCreationAllowance} from 'src/identity/apis/org'
+
+// Types
+import {GetState, RemoteDataState} from 'src/types'
+
+// Utils
+import {reportErrorThroughHoneyBadger} from 'src/shared/utils/errors'
+
+type Actions = OrgCreationAllowancesActions
+
+export const getOrgCreationAllowancesThunk =
+  () => async (dispatch: Dispatch<Actions>, getState: GetState) => {
+    try {
+      dispatch(setOrgCreationAllowancesStatus(RemoteDataState.Loading))
+
+      const allowances = await fetchOrgCreationAllowance()
+
+      dispatch(setOrgCreationAllowances(allowances))
+
+      dispatch(setOrgCreationAllowancesStatus(RemoteDataState.Done))
+    } catch (error) {
+      reportErrorThroughHoneyBadger(error, {
+        name: 'Failed to fetch org creation Allowances',
+        context: {state: getState()},
```

Comment:
Also - this is minor, but I don't think we need all of state here (when I've viewed that on honeybadger it makes it hard to debug). I would just grab so much of state as we need - probably the information from the `identity` object.